docs/example-scenario/apps/app-monitoring.md

```diff
@@ -0,0 +1,149 @@
+
+Runtime monitoring is another easy way to get started. The telemetry that is collected must be controlled through configuration files. For example, you can include runtime methods that enable tools such as [Application Insights Status Monitor][Application Insights Status Monitor] to deploy the SDKs into the correct folder and add the right configurations to begin monitoring.
+
+Like Application Insights, Log Analytics provides tools for [analyzing data across sources][analyzing data across sources], creating complex queries, and [sending proactive alerts][sending proactive alerts] on specified conditions. You can also view telemetry in [the Azure portal][the Azure portal]. Log Analytics adds value to existing monitoring services such as [Azure Monitor][Azure Monitor] and can also monitor on-premises environments.
+
+Both Application Insights and Log Analytics use [Azure Log Analytics Query Language][Azure Log Analytics Query Language]. You can also use [cross-resource queries](https://azure.microsoft.com/blog/query-across-resources/) to analyze the telemetry gathered by Application Insights and Log Analytics in a single query.
+
+Azure Monitor, Application Insights, and Log Analytics all send [alerts](/azure/monitoring-and-diagnostics/monitoring-overview-alerts). For example, Azure Monitor alerts on platform-level metrics such as CPU utilization, while Application Insights alerts on application-level metrics such as server response time. Azure Monitor alerts on new events in the Azure Activity Log, while Log Analytics can issue alerts about metrics or event data for the services configured to use it. [Alerts (Preview)](/azure/monitoring-and-diagnostics/monitoring-overview-unified-alerts) is a new, unified alerting experience in Azure that uses a different taxonomy.
+
+### Alternatives
+
+This article describes conveniently available monitoring options with popular features, but you have many choices, including the option to create your own logging mechanisms. A best practice is to add monitoring services as you build out tiers in a solution. Here are some possible extensions and alternatives:
+
+- Consolidate Azure Monitor and Application Insights metrics in Grafana using the [Azure Monitor Data Source For Grafana][Azure Monitor Data Source For Grafana].
+- Automate monitoring functions using [Azure Automation][Azure Automation].
+- Add communication with [ITSM solutions][ITSM solutions]).
+- Extend Log Analytics with a [management solution][management solution].
+
+### Scalability and availability
+
+This scenario focuses on PaaS solutions for monitoring in large part because they conveniently handle availability and scalability for you and are backed by service-level agreements (SLAs). For example, App Services provides a guaranteed [SLA][SLA] for its availability.
```

Comment:
Should mention scaling considerations for App Insights 
- [Rate limits](https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#application-insights-limits)
- AI will throttle if you go over the limit
- Use [filtering](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-filtering-sampling) or [sampling](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-sampling) to reduce the data rate